libraries/SITL/SIM_Vicon.h

```diff
@@ -75,6 +76,10 @@ class Vicon : public SerialDevice {
 
     void maybe_send_heartbeat();
     uint32_t last_heartbeat_ms;
+
+    uint64_t _pos_delta_prev_time_us;
```

Comment:
let's put a comment beside this variable if possible.

Reply:
added